DATA/production/calib/hmp-pedestals-processing.sh

```diff
@@ -53,35 +52,35 @@ then
     HMP_PED_TAG="Latest"
 fi
 
-#  Here we compose the workflow
-WORKFLOW="o2-dpl-raw-proxy $ARGS_ALL --dataspec \"$PROXY_INSPEC\" --channel-config \"name=readout-proxy,type=pull,"
-WORKFLOW+="method=connect,address=ipc://@$INRAWCHANNAME,rateLogging=1,transport=shmem\" | "
-
-WORKFLOW+="o2-hmpid-raw-to-pedestals-workflow ${ARGS_ALL} --configKeyValues \"$ARGS_ALL_CONFIG\" --fast-decode "
-
+# Compose the specific parameters
+SPEC_PARAM=""
 if [ $HMP_NODCSCCDB_REC == 'false' ];
 then
-  WORKFLOW+="--use-dcsccdb --dcsccdb-uri 'http://alio2-cr1-flp199.cern.ch:8083' --dcsccdb-alivehours 3 "
+  SPEC_PARAM+="--use-dcsccdb --dcsccdb-uri 'http://alio2-cr1-flp199.cern.ch:8083' --dcsccdb-alivehours 3 "
 fi
 if [ $HMP_CCDB_REC == 'true' ];
 then
-  WORKFLOW+="--use-ccdb --ccdb-uri 'http://o2-ccdb.internal' "
+  SPEC_PARAM+="--use-ccdb --ccdb-uri 'http://o2-ccdb.internal' "
 fi
 if [ $HMP_FILES_REC == 'true' ];
 then
-  WORKFLOW+="--use-files "
+  SPEC_PARAM+="--use-files "
 fi
 
-WORKFLOW+="--files-basepath 'HMP' "
-WORKFLOW+="--pedestals-tag ${HMP_PED_TAG} --sigmacut ${HMP_SIGMACUT} | "
+SPEC_PARAM+="--files-basepath 'HMP' "
+SPEC_PARAM+="--pedestals-tag ${HMP_PED_TAG} --sigmacut ${HMP_SIGMACUT}"
 
+#  Here we compose the workflow
+# Start with an empty workflow
+WORKFLOW=
+add_W o2-dpl-raw-proxy "--dataspec \"$PROXY_INSPEC\" --channel-config \"name=readout-proxy,type=pull,method=connect,address=ipc://@$INRAWCHANNAME,rateLogging=1,transport=shmem\""
```

Comment:
```suggestion
add_W o2-dpl-raw-proxy "--dataspec \"$PROXY_INSPEC\" --channel-config \"name=readout-proxy,type=pull,method=connect,address=ipc://@$INRAWCHANNAME,rateLogging=1,transport=shmem\"" "" 0
```
Hi @fapfap69 I have tried the topology generation with the above change and it works. The problem is that by defaul the `add_W` function is adding the `ARGS_ALL_CONFIG ` to the workflow. To disable it one has to add the zero at the end: https://github.com/AliceO2Group/O2DPG/blob/4750f15e3eaca59ac3325d0580fdf8e2c7cb8a99/DATA/common/gen_topo_helper_functions.sh#L223
Sorry I did not spot this earlier